xbmc/Application.cpp

```diff
@@ -4085,16 +4085,24 @@ PlayBackRet CApplication::PlayFile(const CFileItem& item, bool bRestart)
       CJobManager::GetInstance().PauseJobs();
     }
 
-    // don't hold graphicscontext here since player
-    // may wait on another thread, that requires gfx
-    CSingleExit ex(g_graphicsContext);
+    if (item.Exists()) // this is also an implicit call to CWakeOnAccess::WakeUpHost() if enabled
+    {
+      // don't hold graphicscontext here since player
+      // may wait on another thread, that requires gfx
+      CSingleExit ex(g_graphicsContext);
 
-    iResult = m_pPlayer->OpenFile(item, options);
+      iResult = m_pPlayer->OpenFile(item, options);
+    }
+    else
+    {
+      CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Error, 
```

Comment:
Yes good point. 
But if you come from home-screen then the failure will go unnoticed without the toast and IMO that is slightly worse than the duplicate notification?
This is the case also when not using wake-on-access, if an item fails to play from home-screen there is no feedback I believe.
I am prepared to drop the toast if you deem that the correct thing to do, again pls let me know and I will get it done!